src/Button.tsx

```diff
@@ -161,4 +172,8 @@ const useStyles = createStyles(theme => ({
   'color-secondary': {
     backgroundColor: theme.colors.accent,
   },
+  'color-unfilled-with-border': {
+    backgroundColor: theme.colors.background,
+    border: `2px solid ${theme.colors.onSurface}`,
```

Comment:
move to `rounded`

Reply:
i'm going to keep the color of the border here but update to just be `unfilled` b/c to me the outside border color should be tied to the inside color